### PR TITLE
Asking for better config template

### PR DESCRIPTION
**Pull request by shushui**
Out of trying out the app, I am guessing the format of the configuration.
(I am new to Python, so maybe I also miss some common-python-knowledge I don't know of)

### CODE REVIEW

File: semantle.cfg.format [hunk 1]
```diff
@@ -1,2 +1,9 @@
 #!/usr/bin/env python
 """ remove .format, and add configuration secretes here """
+easter_eggs= "42"
+api_key="dd"
+quotes="dd"
+""" In your mongodb there should be a database named Semantle and a collection named word2vec2 in order to populate"""
+mongo="mongodb+srv://42"
+redis="42"
```
Comment (ishefi) [inline semantle.cfg.format:8]: should be a redis url string 
```
redis = 'redis://USERNAME:PASSWORD@HOST:PORT/'
```

File: semantle.cfg.format [hunk 2]
```diff
@@ -1,2 +1,9 @@
 #!/usr/bin/env python
 """ remove .format, and add configuration secretes here """
+easter_eggs= "42"
+api_key="dd"
+quotes="dd"
```
Comment (ishefi) [inline semantle.cfg.format:5]:
```python
quotes: [Tuple[str,str,str,str]] = [
(
    QOUTE,
    QOUTEE,
    SOURCE,
    URL
)
```

File: semantle.cfg.format [hunk 3]
```diff
@@ -1,2 +1,9 @@
 #!/usr/bin/env python
 """ remove .format, and add configuration secretes here """
+easter_eggs= "42"
```
Comment (ishefi) [inline semantle.cfg.format:3]:
```python
easter_eggs: Dict[str,str] = {EASTER: EGG}
```

**Review by ishefi — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ You can run the game with:
 ```commandline
 PORT=<PORT> python app.py
 ```
-Configurations can be set in `config.py` (not part of the repo as it contains secrets).
+Configurations should be set in a `semantle.cfg` file, following the example `semantle.cfg.format`.
```
Comment (ishefi) [inline README.md:21]: it's actually _can_ and not _should_, as you can pass the entire configuration through env var `HS_CONFIG`

**Comment by Iddoyadlin** [comment]
there is now a better template (see config.format.yaml)

**Pull request closed without merging**